frontend/src/components/activePage/Instances/InstanceContent/ManagedInstanceHeading.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+import { Typography } from 'antd';
+
+export interface IManagedInstanceHeadingProps {
+  tenantId: string;
+  tenantDisplayName: string;
+  displayName: string;
+}
+
+const ManagedInstanceHeading: FC<IManagedInstanceHeadingProps> = ({
+  ...props
+}) => {
+  const { tenantId, tenantDisplayName, displayName } = props;
+  const { Text } = Typography;
```

Comment:
I believe you can import directly the `Text` component

Reply:
`import { Text } from 'antd';`

Does not work so I am afraid not :/